bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -52,6 +52,8 @@
 
     private static Logger logger = LoggerFactory.getLogger(HttpUtil.class);
 
+    private static final long DEFAULT_TIMEOUT = 5000;
```

Comment:
Given the value I would hope it's seconds, can you make it clear by changing the name for example `DEFAULT_TIMEOUT _MS`

Alternative what would you think about accepting a Duration, see for example the following fragment:

```java
downloadImage(url, Duration.ofSeconds(15));

// or with a static import
downloadImage(url, ofSeconds(15));
```

Reply:
I will rename the constant.